src/SfxWeb/src/app/modules/event-store/event-store/event-store.component.ts

```diff
@@ -274,13 +281,25 @@ export class EventStoreComponent implements OnInit, OnDestroy {
                                     if (!inputEvent.related) {
                                         inputEvent.related = [];
                                     }
-                                    if(Utils.result(iterEvent, relevantEventType.result)) {
-                                        action = "Action: " + Utils.result(iterEvent, relevantEventType.result) + "<br/><br/>";
-                                    }
-                                    iterEvent.reasonForEvent = action;
                                     inputEvent.related.push(iterEvent);
                                     addedEvents.push(iterEvent);
                                 }
+                            } else if(relevantEventType.eventType == "self") {
```

Comment:
wouldnt this trigger for every event?

Reply:
so should we switch the iterevent and relevantevetntype loops?